packages/amplify-cli/src/pull-backend.ts

```diff
@@ -33,7 +33,7 @@ export async function pullBackend(context, inputParams) {
 
   await initializeEnv(context);
   ensureBackendConfigFile(context);
-  await postPullCodeGenCheck(context);
+  await postPullCodegen(context);
```

Comment:
What was postPullCodeGenCheck() doing before and what is postPullCodegen() doing differently?

Reply:
I'm not sure why it was there before. All it did was print out an empty line to the console

Reply:
Haha, okay.